glean-core/python/glean/config.py

```diff
@@ -3,9 +3,49 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
+import dataclasses
+from typing import Optional
+
+
+# The default server pings are sent to
+DEFAULT_TELEMETRY_ENDPOINT = "https://incoming.telemetry.mozilla.org"
+
+
+# The default UserAgent string
+def get_default_user_agent():
+    # Need to use a function here to avoid a cyclical import problem
+    from glean import __version__
+
+    return f"Glean/{__version__} (Python)"
+
+
+# The default number of events to store before sending
+DEFAULT_MAX_EVENTS = 500
```

Comment:
Does it make sense to move all the default/constants at the beginning of the file?

Reply:
Technically, these were all "constants", just one had to have deferred evaluation.  But I had to refactor this anyway to make the documentation generator happy :)